shuup/utils/http.py

```diff
@@ -9,6 +9,9 @@
 
 import requests
 
+from django.conf import settings
+from ipware.ip import get_ip, get_real_ip
+
```

Comment:
`requests` is also third-party, so these new lines shouldn't be separated from it by blank line.